std/math.d

```diff
@@ -3601,6 +3609,11 @@ real log2(real x) @safe pure nothrow @nogc
 ///
 @system unittest
 {
+    assert(approxEqual(log2(1024.0L), 10));
```

Comment:
equalsDigit is private, that's why I copied the test to a separate unittest and inserted approxEqual instead